File: scan/src/main/java/com/yunxiao/scan/ScanMethodUtil.java
package com.yunxiao.scan;

import android.Manifest;

import java.util.Arrays;

/**
 * @author Created by rongkaifang
 * @date Created at 4:50 PM on 2021/3/4
 * @description
 */
public class ScanMethodUtil {

    final static String[] permissions = new String[]{Manifest.permission.CAMERA};
    final static int REQUESTPERMISSION = 0x00;
    final static int REQUESTACTIVITYCODE = 0x01;

}

File: scan/src/main/java/com/yunxiao/scan/ScanMethodUtil.kt
package com.yunxiao.scan

import android.Manifest

/**
 * @author Created by rongkaifang
 * @date Created at 10:33 AM on 2020/11/13
 * @description
 */
class ScanMethodUtil {
    companion object {
        var permissions = arrayOf(Manifest.permission.CAMERA)
        const val REQUESTPERMISSION = 0x00
        const val REQUESTACTIVITYCODE = 0x01

    }
}